lib/querystring.js

```diff
@@ -174,11 +175,12 @@ function qsEscape(str) {
     }
     // Surrogate pair
     ++i;
-    var c2;
-    if (i < str.length)
-      c2 = str.charCodeAt(i) & 0x3FF;
-    else
-      throw new URIError('URI malformed');
```

Comment:
I'm not sure if this should to be changed to use internal/errors. [`URIError`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError) seems fine to me.

Reply:
I would definitely recommend to change this to use `internal/errors` but it should be kept as URIError as I mentioned above (https://github.com/nodejs/node/pull/15565#discussion_r140633423).